deploy/helm-chart/values.yaml

```
@@ -48,6 +48,9 @@ nginxGateway:
   ## The lifecycle of the nginx-gateway container.
   lifecycle: {}
 
+  # extraVolumeMounts are the additional volume mounts for the nginx-gateway container.
```

nit:
```suggestion
  ## extraVolumeMounts are the additional volume mounts for the nginx-gateway container.
```

sorry, just noticed this.